docs/en/dql-parser.rst

```diff
@@ -7,6 +7,8 @@ like the following:
 
```

Comment:
This the sentence above says it looks "something like the following", I think it's OK, to make the ORM Lexer look more modern than it currently is.